src/components/Accordion/AccordionHeader.tsx

```diff
@@ -1,27 +1,29 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import React, { FC, ReactElement, cloneElement, isValidElement } from 'react';
+import React, { ReactElement, cloneElement, isValidElement } from 'react';
 import { IconSize } from '@foundation/Icon/IconSize';
 import { IconProps } from '@foundation/Icon/IconProps';
 import { merge } from '@utilities/merge';
 import { AccordionHeaderIconSize, AccordionHeaderProps } from './types';
 import { AccordionHeaderIcon } from './AccordionHeaderIcon';
 
-export const AccordionHeader: FC<AccordionHeaderProps> = ({
+export const AccordionHeader = ({
     isOpen = false,
     decorator,
     disabled = false,
     children,
     size = AccordionHeaderIconSize.Medium,
     bold = true,
     type,
-    as: Heading = 'div',
-}) => {
+    as: Heading = 'span',
+}: AccordionHeaderProps) => {
     const icon = <AccordionHeaderIcon isOpen={isOpen} disabled={disabled} size={size} type={type} />;
     return (
-        <div data-test-id="accordion-header" className="tw-px-8 tw-py-6">
-            <header
+        <span data-test-id="accordion-header" className="tw-block tw-px-8 tw-py-6">
+            <span
```

Comment:
This is not correct HTML, `span > div` is not allowed

Reply:
The children prop in `AccordionHeaderProps` is defined as string as you can see here
```
export type AccordionHeaderProps = {
    decorator?: ReactNode;
    children: string;
    disabled?: boolean;
    isOpen: boolean;
    /** @deprecated use headerComponent prop for custom header styles */
    size?: AccordionHeaderIconSize | FieldsetHeaderSize;
    /** @deprecated use headerComponent prop for custom header styles */
    bold?: boolean;
    /** @deprecated Icon type will be fixed to FieldsetHeaderType.Accordion.
     Use headerComponent prop for custom header styles */
    type?: FieldsetHeaderType;
    /** @deprecated use headerComponent prop for custom header styles */
    as?: keyof JSX.IntrinsicElements;
};
```

Reply:
I changed to span because on the `AccordionItem` the header is rendered inside a `button` and `button > div` is also not valid HTML

Reply:
https://github.com/Frontify/fondue/blob/beta/src/components/Accordion/Accordion.tsx#L37